mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/json/MessageToElasticSearchJsonTest.java

```diff
@@ -63,11 +69,22 @@
     public static final MessageUid UID = MessageUid.of(25);
     public static final Charset CHARSET = Charsets.UTF_8;
 
+    private TextExtractor textExtractor;
+
     private Date date;
     private PropertyBuilder propertyBuilder;
 
+    @Rule
+    public TikaContainer tika = new TikaContainer();
+
     @Before
     public void setUp() throws Exception {
+        tika.start();
+        textExtractor = new TikaTextExtractor(new TikaHttpClientImpl(TikaConfiguration.builder()
```

Comment:
Could be great to put that in a rule too so we will just need to do `tikaRule.getTextExtractor`